include/wx/gtk/menuitem.h

```diff
@@ -39,6 +39,10 @@ class WXDLLIMPEXP_CORE wxMenuItem : public wxMenuItemBase
     GtkWidget *GetMenuItem() const { return m_menuItem; }
     void SetGtkLabel();
 
+#if wxUSE_ACCEL
+    void SetGtkExtraAccels();
```

Comment:
We use `GTK` prefix by convention for wxGTK-specific methods that are not part of the public API (also `MSW`, `OSX`, ...).
```suggestion
    void GTKSetExtraAccels();
```